builder/ebs/acceptance/utils.go

```diff
@@ -0,0 +1,22 @@
+package amazon_acc
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func GenerateSSHPrivateKeyFile() (string, error) {
```

Comment:
chatted about moving this out of the utils file cause of failures with the windows acceptance test with @lbajolet-hashicorp in a call

Reply:
For visibility, we're not moving this away, but we add extra checks to avoid running this unless we're running the acceptance tests here.
Hopefully, the last commit should fix this issue.